tutorial1_intro/README.md

```diff
@@ -9,15 +9,33 @@ The goals for this tutorial are:
 - to introduce [d3-selections](https://bost.ocks.org/mike/selection/) and [d3 data binding](https://observablehq.com/@d3/selection-join).
 - to give the tools to create a simple [HTML table](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table).
 
-## Assignment:
+## Serve and Set Up:
 
-1. Go through the [demo code](demo/) understand how the code works
-2. Replicate the process for **your own dataset** in the files present in the root of this directory ([index.html](index.html), [style.css](style.css), [main.js](main.js))
-3. Use the Github workflow to deploy your work to **your fork** of the course repository
-4. Post the link to your deployed example on the Commons. **//TODO: check this @Ellie**
-   (Note: your link should look something like: `https://YOUR_USERNAME.github.io/Interactive-Data-Vis-Sp2020/tutorial1_intro/`)
+To begin working on your tutorial assignment on your local machine, use your terminal (or git bash, on PC) to navigate to the _root directory_ of your fork. Your path may look different, but the command will look something like this:
+
+```
+cd Documents/My\ Repositories/Interactive-Data-Vis-Sp2020/
+```
+
+From this level, (_not from the tutorial folder_), you should run `serve`. When you open  `http://localhost:5000/`, you should see your repository's folder structure. Click into this week's tutorial. The file showing is your `index.html` file. If you want to view the demo file, you have to change your URL path. For example: `http://localhost:5000/tutorial1_intro/demo`.
+
+## Week 1 Assignment:
+
+### System Preparation
+1. Complete the tech set up, including [git setup](../GIT_SETUP.md) (forking, cloning, setting upstream, etc) and getting a [basic server](../BASIC_SERVER.md) up and running
+2. Review the [demo code](demo/) and understand how the code works
+
+### Code Build
+3. Replicate the process for **your own dataset** in the files present in the root of this directory ([index.html](index.html), [style.css](style.css), [main.js](main.js))
+4. Update the logic determining cell classname to apply to your dataset
```

Comment:
Suggestion: 

> Update the table's conditional formatting to highlight something meaningful for your new dataset. This may include changing the logic the [class](https://github.com/d3/d3-selection#selection_classed), adding another [attribute](https://github.com/d3/d3-selection#selection_attr), or directly augmenting the cell's [style](https://github.com/d3/d3-selection#selection_style).